pyanaconda/packaging/dnfpayload.py

```diff
@@ -113,27 +113,47 @@ def paced_fn(self, *args):
         return fn(self, *args)
     return paced_fn
 
-def _pick_mpoint(df, download_size, install_size):
+def _pick_mpoint(df, download_size, install_size, download_only):
     def reasonable_mpoint(mpoint):
         return mpoint in DOWNLOAD_MPOINTS
 
     requested = download_size
     requested_root = requested + install_size
     root_mpoint = pyanaconda.iutil.getSysroot()
-    sufficients = {key : val for (key, val) in df.items()
-                   # for root we need to take in count both download and install size
-                   if ((key != root_mpoint and val > requested)
-                   or val > requested_root) and reasonable_mpoint(key)}
     log.debug('Input mount points: %s', df)
     log.debug('Estimated size: download %s & install %s', requested,
               (requested_root - requested))
-    log.info('Sufficient mountpoints found: %s', sufficients)
+
+    # Find sufficient mountpoint to download and install packages.
+    sufficients = {key : val for (key, val) in df.items()
+                   if ((key != root_mpoint and val > requested) or val > requested_root)
+                      and reasonable_mpoint(key)}
+
+    # If no sufficient mountpoints for download and install was found and we are looking
+    # for download mountpoint only, ignore install size and try to find mountpoint just
+    # to download packages. This fallback is required when user skipped space check.
+    if not len(sufficients) and download_only:
```

Comment:
I find ``if not sufficients`` better readable and understandable than testing the length. If you want to use the length in the condition, compare it to ``0``.